### DISCUSSION TITLE
Is there a way to break the internal/Child pipeline conditionally?

### DISCUSSION THREAD

**Discussion by vkydhyani** [apache/hop#4740] [General]
I have a workflow which internally calls some common pipelines and calls the API with subscription key. Now what is happening even if the subs key is wrong, sometime plugin does not break.

If this is not behaving the way we would like, then we want to conditionally control this.
Is there a way to break the internal pipeline conditionally? If no response from Rest Client. I want plugin to raise an error.

**Comment by bamaer** [comment]
you should be able to do this by wrapping your business logic in a workflow or pipeline, run that through the [Repeat](https://hop.apache.org//manual/latest/workflow/actions/repeat.html) action, which allows you to break the loop by e.g. setting a variable on a given condition.

**Comment by hansva** [comment]
You can also use the abort transform, this can be used to stop a pipeline when it receives a row.
